src/components/Inputs/PasswordInput.js

```diff
@@ -23,12 +24,7 @@ const PasswordInput = () => {
           secureTextEntry={!passwordVisible}
         />
         <TouchableOpacity onPress={() => setPasswordVisible(!passwordVisible)}>
```

Comment:
Use `TouchableWithoutFeedback` for this instead, too many micro-interactions on this element make it janky.